Documentation/ceph-nfs-crd.md

```diff
@@ -4,19 +4,22 @@ weight: 3100
 indent: true
 ---
 
-# Ceph NFS Gateway CRD
+# Ceph NFS Server CRD
 
 ## Overview
+Rook allows exporting NFS shares of the filesystem or object store through the CephNFS custom
```

Comment:
For the "object store" are you referring to rgw-nfs? If so, it's not using CephFS but simply RGW.

Reply:
I will clarify by changing this to `CephFilesystem` and `CephObjectStore`. I'm not sure what `rgw-nfs` is?